src/main/java/io/buildpacks/example/sample/CalculatorService.java

```diff
@@ -2,25 +2,82 @@
 
 import org.springframework.stereotype.Service;
 
+/**
+ * Service class providing basic calculator operations.
+ */
 @Service
 public class CalculatorService {
```

Comment:
8. **Dependency Injection**:
   - While not directly related to the code provided, consider using dependency injection for potential dependencies in the `CalculatorService` class to improve testability and maintainability.